contracts/metatx/MinimalForwarder.sol

```diff
@@ -69,4 +99,51 @@ contract MinimalForwarder is EIP712 {
 
         return (success, returndata);
     }
+
+    function _verify(ExpiringForwardRequest memory req, bytes memory signature) public view virtual returns (bool) {
+        bool expired = block.timestamp >= req.deadline;
+        return
+            getNonce(req.from) == req.nonce && // Signed nonce corresponds with current nonce
+            _signer(req, signature) == req.from && // From is signer
+            !expired;
+    }
+
+    function _signer(ForwardRequest memory req, bytes memory signature) private view returns (address) {
```

Comment:
this function is currently unused, because you rewrite everything as `ExpiringForwardRequest`.

Reply:
Used now.